src/python/foglamp/core/api/scheduler.py

```diff
@@ -372,8 +372,8 @@ def _extract_args(data, curr_value):
         curr_value['schedule_process_name'] = sch.process_name
         curr_value['schedule_name'] = sch.name
         curr_value['schedule_type'] = sch.schedule_type
-        curr_value['schedule_repeat'] = sch.repeat
-        curr_value['schedule_time'] = sch.time
+        curr_value['schedule_repeat'] = sch.repeat.total_seconds() if sch.repeat else 0
+        curr_value['schedule_time'] = (sch.time.hour * 60 * 60 + sch.time.minute * 60 + sch.time.second) if sch.time else 0
```

Comment:
is `server.Server.scheduler.` okay to receive time and repeat? don;t we need to convert to python timedelta struct?

Reply:
as this `curr_value` is more api to core scheduler communication; not from API to UI

Reply:
yes, it is more core scheduler -> api communication. Because of this, we do not need to convert repeat and time values.